core/templates/components/forms/custom-forms-directives/object-editor.directive.ts

```diff
@@ -40,7 +40,7 @@ angular.module('oppia').directive('objectEditor', [
         initArgs: '=',
         isEditable: '@',
         objType: '@',
-        schema: '=',
+        getSchema: '&schema',
```

Comment:
Just to double-check -- you tested this manually right?

Reply:
Yes, I did